lib/perm/is-ticketing-admin.js

```diff
@@ -20,7 +22,45 @@ function isTicketingAdmin (args, cb) {
 
   var isTicketingAdmin = false;
   //  Could also check the opposite way, from child to Parent
-  seneca.act({role: 'cd-dojos', cmd: 'load_usersdojos', query: { userId: userId, dojoId: dojoId }},
+  function checkPrerequisites (wfCb) {
+    function getDojoFromEvent (_wCb) {
+      if (eventId) { // We need to get the dojoId associated
+        seneca.act({role: 'cd-events', cmd: 'getEvent', id: eventId}, function (err, event) {
+          if (err) return cb(err);
+          if (event.dojoId) {
+            dojoId = event.dojoId;
+            return _wCb();
+          }
+        });
+      } else {
+        return _wCb();
+      }
+    }
+    function getEventFromSession (_wCb) {
```

Comment:
Flip order of function definitions. Got confused at first because they're in the opposite order of how they're run.